lab_3_generate_by_ngrams/main.py

```diff
@@ -221,6 +323,18 @@ def generate_next_token(self, sequence: tuple[int, ...]) -> Optional[dict]:
 
         In case of corrupt input arguments, None is returned
         """
+        if not isinstance(sequence, tuple) or\
+                len(sequence) == 0:
```

Comment:
Empty sequences correspond to False as is, there is no need in additional check of equivalence

Reply:
Fixed